README.md

```diff
@@ -1,2 +1,3 @@
 # git-demo
 to demonstrate how to use git and GitHub
+Tianshu Zhang https://github.com/tianshu-zhang
```

Comment:
Could you please add a `-` so this becomes a list item?

Also, please make your name into a link that goes to that URL. The URL should not be visible.

Reply:
Hi Professor Baumer, I've added the `-` and modified my name into a hyperlink. Let me know if other modifications are needed. Thanks!